src/Libm.jl

```diff
@@ -1,10 +1,7 @@
-
 include("Musl.jl")
 include("Sleef.jl")
```

Comment:
What did you try? It's not right for a package to be defining 2 top-level modules, and it causes precompilation issues. https://github.com/tanmaykm/ProtoBuf.jl/issues/77